view/View.class.php

```diff
@@ -139,8 +139,11 @@ public static function compileCss()
       $scssCompiler->setLineNumberStyle(Leafo\ScssPhp\Compiler::LINE_COMMENTS);
     }
 
-    $css = $scssCompiler->compile(file_get_contents(self::SCSS_DIR . '/all.scss'));
-    file_put_contents(self::CSS_DIR . '/all.css', $css);
+    $all_css = $scssCompiler->compile(file_get_contents(self::SCSS_DIR . '/all.scss'));
+    file_put_contents(self::CSS_DIR . '/all.css', $all_css);
+
+    $yt2_css = $scssCompiler->compile(file_get_contents(self::SCSS_DIR . '/yt2.scss'));
```

Comment:
Please call this just "youtube" instead of "yt2". It was initially called yt2 to prevent a conflict with the existing youtube page. Any references to yt2 should just be "youtube" at this point.